### PR TITLE
Stripe checkout v3

### PR DESCRIPTION
**Pull request by ronix-webriq**
Getting List of Products & Prices from Stripe using API endpoint from app/web **dxpstudio** and made a checkout button for variant A.

### CODE REVIEW

File: component/sections/pricing/index.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
 import React from "react"
 import dynamic from "next/dynamic"
+// import {Elements} from '@stripe/react-stripe-js';
+// import {loadStripe} from '@stripe/stripe-js';
```
Comment (webriq) [inline component/sections/pricing/index.js:4]: Let's totally remove this commented lines.

File: component/sections/pricing/index.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 
-function Pricing({ data }) {
-  const variant = data?.variants?.variant
+const {
+  NEXT_PUBLIC_DXP_STUDIO_ADDRESS
```
Comment (webriq) [inline component/sections/pricing/index.js:7]: please change to `NEXT_PUBLIC_APP_URL`

File: component/sections/pricing/index.js [hunk 3]
```diff
@@ -14,9 +20,19 @@ function Pricing({ data }) {
     monthlyBilling: data?.variants?.[variant]?.monthlyBilling,
     banner: data?.variants?.[variant]?.banner,
     form: data?.variants?.[variant]?.form,
+    stripePKey: data?.variants?.[variant]?.stripeAccount?.stripePKey,
```
Comment (webriq) [inline component/sections/pricing/index.js:23]: Not sure but do we still need this `stripePKey` ?

**Review by webriq — CHANGES_REQUESTED** [review]

File: component/sections/pricing/index.js (hunk 3 above)
Comment (ronix-webriq) [reply component/sections/pricing/index.js:23]: Yes, for checkout button.

File: component/sections/pricing/variant_d.js [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
+import axios from 'axios'
+
+// const cardElementOptions ={
+//   style: {
+//       base: {         
+//           padding: '10px'
+//       }
+//   },
+//   hidePostalCode: true
+// };
```
Comment (webriq) [inline component/sections/pricing/variant_d.js:14]: Let's remove this commented lines. They seemed to be not anymore of relevance.

File: component/sections/pricing/variant_d.js [hunk 5]
```diff
@@ -1,6 +1,17 @@
 import { urlFor } from "lib/sanity";
 import React from "react";
 import WebriQForm from "@webriq/gatsby-webriq-form";
+// import {CardElement, useStripe, useElements} from '@stripe/react-stripe-js';
```
Comment (webriq) [inline component/sections/pricing/variant_d.js:4]: Let's remove this

File: component/sections/pricing/variant_d.js [hunk 6]
```diff
@@ -151,20 +179,20 @@ function VariantD({
                         By signing up, you agree to our{" "}
                         <a
                           className="text-webriq-darkblue font-bold hover:text-webriq-darkblue"
-                          href="#"
+                          href="/terms-data-policy"
```
Comment (webriq) [inline component/sections/pricing/variant_d.js:182]: Can we customize this so they can paste a URL or an internal link?

File: component/sections/pricing/variant_d.js [hunk 7]
```diff
@@ -151,20 +179,20 @@ function VariantD({
...
                         >
                           Terms,
                         </a>
                         <a
                           className="text-webriq-darkblue font-bold hover:text-webriq-darkblue"
-                          href="#"
+                          href="/terms-data-policy"
```
Comment (webriq) [inline component/sections/pricing/variant_d.js:188]: Can we customize this so they can paste a URL or an internal link?

File: component/sections/pricing/variant_d.js [hunk 8]
```diff
@@ -151,20 +179,20 @@ function VariantD({
...
                         >
                           Data Policy
                         </a>{" "}
                         and{" "}
                         <a
                           className="text-webriq-darkblue font-bold hover:text-webriq-darkblue"
-                          href="#"
+                          href="/cookies-policy"
```
Comment (webriq) [inline component/sections/pricing/variant_d.js:195]: Can we customize this so they can paste a URL or an internal link?

**Review by webriq — CHANGES_REQUESTED** [review]

**Pull request closed without merging**